install/kubernetes/cilium/values.yaml

```diff
@@ -1326,6 +1326,8 @@ hubble:
 
     # -- hubble-ui service configuration.
     service:
+      # -- Annotations to be added for the Hubble UI service
+      annotations: {}
```

Comment:
Please report this change to `install/kubernetes/cilium/values.yaml.tmpl` and then run:
```
make -C install/kubernetes
```